src/keymap.rs

```diff
@@ -13,6 +14,58 @@ use crate::tty::{RawReader, Term, Terminal};
 /// The number of times one command should be repeated.
 pub type RepeatCount = usize;
 
+/// Custom dynamic action
+#[derive(Clone)]
+pub struct Action {
+    pub name: String,
```

Comment:
Can you say what this name is used for? It's not clear to me.

Reply:
It is used only for tracing / debugging.